projects/memapCore/src/main/java/memap/messages/planning/DemandMessage.java

```diff
@@ -12,6 +12,11 @@
 	private double[] demandForecast; // that certainly includes both heat and electricity
 	public String forecastType;
 	public NetworkType networkType;
+	
+	public double[] varNetworkBuyCostEUR;
+	public double[] varNetworkSellCostEUR;
```

Comment:
this is the same as the prices, right? We can use that later to assign individual prices for each building in the simulation later on.